### ISSUE TITLE
Group tomatoes in cycles in the start focusing tab and take a break tab

### ISSUE THREAD

**Issue by hongtaobai** [schmich/marinara#320]
Currently that tab shows how many tomatoes completed today in total.

Beyond that, this update will make it easy to tell 
1. How many cycles completed today
2. Where I am in current cycle

**Comment by eneda8** [comment]
agreed, and it also resets at midnight. if you're working at 11pm, you lose track of your previous tomatoes. it would be great if it could show you current session (current cycle)
